apps/meteor/client/views/room/hooks/useUserInfoActions/useUserInfoActions.ts

```diff
@@ -14,50 +16,100 @@ import { useMuteUserAction } from './actions/useMuteUserAction';
 import { useRedirectModerationConsole } from './actions/useRedirectModerationConsole';
 import { useRemoveUserAction } from './actions/useRemoveUserAction';
 
+export type UserInfoAction = {
+	content: ReactNode;
+	icon?: ComponentProps<typeof Icon>['name'];
+	onClick: () => void;
+	type?: string;
```

Comment:
Maybe type of `type` should be - 

`type?: 'communication' | 'privileges' | ....` for better type safety, maybe we can also create an enum for keeping these types consistent.